packages/sw-precaching/src/lib/controllers/revisioned-cache-manager.js

```diff
@@ -53,6 +54,21 @@ class RevisionedCacheManager extends BaseCacheManager {
   addToCacheList({revisionedFiles} = {}) {
     assert.isInstance({revisionedFiles}, Array);
     super._addEntries(revisionedFiles);
+
+    const urlsWithoutRevisionFields = revisionedFiles
+      .filter((entry) => typeof entry === 'string');
+    if (urlsWithoutRevisionFields.length > 0) {
+      logHelper.debug({
+        that: this,
+        message: `Some precache entries don't include a separate revision
```

Comment:
I've cut down on the "above the fold" message and tried to include as much helpful context as I could in the `data` fields. PTAL.